modules/core/src/controllers/orbit-controller.js

```diff
@@ -2,6 +2,15 @@ import {clamp} from 'math.gl';
 import Controller from './controller';
 import OrbitViewport from '../deprecated/orbit-viewport';
 import assert from '../utils/assert';
+import LinearInterpolator from '../transitions/linear-interpolator';
+import {TRANSITION_EVENTS} from './transition-manager';
+
+const LINEAR_TRANSITION_PROPS = {
+  transitionDuration: 300,
+  transitionEasing: t => t,
+  transitionInterpolator: new LinearInterpolator(),
```

Comment:
I think the default props include longitude/latitude?

Reply:
Right, all we need is to transition 'zoom' on double-tap, we can update this to `zoom` only and also update `_getTransitionProps` to return these props for double-tap only?

Reply:
What about keyboard navigation?